pypasta/Repository/Mbox.py

```diff
@@ -322,17 +320,11 @@ def add_mbox(self, listname, f_mbox_raw):
         self.raw_mboxes.append((listname, f_mbox_raw))
         f_mbox_index = os.path.join(self.d_index, 'raw.%s' % listname)
         self.load_index(f_mbox_index)
-        log.info('  ↪ loaded mail index for %s: found %d mails' % (listname, len(index)))
+        log.info('  ↪ loaded mail index for %s: found %d mails' % (listname, len(self.index)))
```

Comment:
Oh, that's even a bug fix for a bug that I introduces earlier, right? Will take this to the bug-introducing patch and fix it there. Nice catch! Thanks!

Reply:
Ah crap, I introduced a bigger problem with load_index, I need to reconsider it. I'll need to fix that first. I didn't respect that load_index will need to support sequential index loading for raw mboxes.

Reply:
Yup, realised that you must have missed that